dvc/system.py

```diff
@@ -12,7 +12,7 @@
 if (
     platform.system() == "Windows"
```

Comment:
@pared, what happens if we change this to following?
```suggestion
    sys.platform == "win32"
```

Does that work?

Reply:
But I am fine with this change as well. 🙂

Reply:
it does, will change the PR